test/integration/integration_test.go

```diff
@@ -301,6 +317,13 @@ func ensureTestDirEmpty() {
 	}
 }
 
+func cleanTestDirConfiguration() {
+	configPath := filepath.Join(testDir, "config")
+	addonsPath := filepath.Join(testDir, "addons")
+	os.RemoveAll(configPath)
+	os.RemoveAll(addonsPath)
```

Comment:
@agajdosi We should check if `os.RemoveAll` returns any error. Otherwise it will fail saliently and it might effect the tests.

Reply:
@LalatenduMohanty Good point, will add the check.